src/components/Wallet.vue

```diff
@@ -55,18 +59,25 @@ import type { WalletPermissions } from '../consts';
     WalletAssets,
     WalletActivity,
     QrCodeScanButton,
+    WalletTransactionDetails,
   },
 })
 export default class Wallet extends Mixins(TranslationMixin, QrCodeParserMixin) {
   readonly WalletTabs = WalletTabs;
 
   @state.router.currentRouteParams private currentRouteParams!: Record<string, Nullable<WalletTabs>>;
   @state.settings.permissions permissions!: WalletPermissions;
+  @getter.transactions.selectedTx selectedTransaction!: HistoryItem;
+  @mutation.transactions.setTxDetailsId setTxDetailsId!: (id: Nullable<string>) => void;
```

Comment:
1. `private setTxDetailsId` cuz you use it in the current class
2. `id: Nullable<string>` -> `id: string`. I suggest to add another mutation like `resetTxDetailsId()`.